app/assets/javascripts/discourse/app/lib/hashtag-autocomplete.js

```diff
@@ -221,19 +221,13 @@ function _searchRequest(term, contextualHashtagConfiguration, resultFunc) {
 }
 
 function _findAndReplaceSeenHashtagPlaceholder(
-  slug,
+  slugRef,
   contextualHashtagConfiguration,
   hashtagSpan
 ) {
   contextualHashtagConfiguration.forEach((type) => {
-    // remove type suffixes
```

Comment:
This was actually not correct since the `seenHashtags` has always used the `.ref`, see https://github.com/discourse/discourse/blob/4026b6a3c54bb267e5e6eafa6116ab60d2ee8993/app/assets/javascripts/discourse/app/lib/hashtag-autocomplete.js#L89-L89